docs/src/main/sphinx/connector/hive.rst

```diff
@@ -110,18 +109,18 @@ Hive views
 Hive views are defined in HiveQL and stored in the Hive Metastore Service. They
 are analyzed to allow read access to the data.
 
-The Hive connector includes support for reading Hive views with three different
+The Hive connector includes support for reading hive views with three different
```

Comment:
Uppercase Hive everywhere.

Reply:
Also since one of the modes is `Disabled` the better text could be `includes support for handling Hive views with three different`? WDYT @mosabua @Jessie212 ?

Reply:
I think the current version is fine